test/views/html/RevaluationSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
this will be updated in another journey, correct?  It is still referencing gmp_main_old

Reply:
Will be covered by other tickets